app/features/config/SelectWowDir.tsx

```diff
@@ -20,6 +20,21 @@ function getDefaultPath() {
   return '';
 }
 
+function validateSelectedPath(path: String) {
+  const fs = require('fs');
+  const addonFolder = path + '/Interface/AddOns/';
```

Comment:
Use Node Path for this, or else it will not work on windows. https://nodejs.org/api/path.html#path_path_join_paths